### DISCUSSION TITLE
Why can't I hear anything on downloaded bird sound clip?

### DISCUSSION THREAD

**Discussion by mhaeberli** [mcguirepr89/BirdNET-Pi#906] [Q&A]
For example, it identified a number of 'Annas Hummingbird' events.  The spectrogram looked plausibly like the clicking sound I hear when a woodpecker visits.  But I download the sound sample, and I hear nothing.

**Comment by matthew73210** [comment]
Try turning up the volume. I find that I have to up the volume to max just to hear anything.
